core/sys/orca/orca.odin

```diff
@@ -1607,7 +1772,7 @@ mouse_state :: struct {
 	wheel: vec2,
 	using _: struct #raw_union {
 		buttons: [5]key_state,
-		using _: struct {
+		_: struct {
```

Comment:
Should `using` really be removed here?

Reply:
mustve been inserted manually and the generator removed it, i only auto gen for raw unions cases. will add it